build.gradle

```diff
@@ -126,13 +126,12 @@ allprojects { subproj ->
 	}
 
 	tasks.withType(JavaCompile) {
-		sourceCompatibility = rootProject.sourceCompatibility
-		targetCompatibility = rootProject.targetCompatibility
```

Comment:
I think we should keep specifying source and target.

Reply:
`javac` does not allow this:

```
Execution failed for task ':junit-platform-commons:compileJava'.
> option -source cannot be used together with --release
```

When added to `compilerArgs` via:

```
	// See: https://docs.oracle.com/javase/9/tools/javac.htm#JSWOR627
	compileJava.options*.compilerArgs = [
		'--release', rootProject.javacRelease,
		'-source', rootProject.javacRelease,
		'-target', rootProject.javacRelease,
...
```

Reply:
But our build refers to `compileJava.sourceCompatibility` here and there. Like in `cloverInstrumentJava` around line 224: `source: compileJava.sourceCompatibility) {`.